packages/features/bookings/Booker/components/DatePicker.tsx

```diff
@@ -35,6 +36,21 @@ export const DatePicker = ({
     shallow
   );
   const nonEmptyScheduleDays = useNonEmptyScheduleDays(schedule?.data?.slots);
+  const browsingDate = month ? dayjs(month) : dayjs().startOf("month");
```

Comment:
What is the reason for having a fallback from `undefined` to `dayjs().startOf("month")`?

Reply:
Not a side effect, but after passing this to the DatePickerComponent, we perform a similar check if browsingDate is undefined. That's why I changed it here

Reply:
If it's not really needed, I think it would be better to undo this change. 

We should fix it by identifying if DatePicker must be passed an undefined value or not. If  passing undefined is a way to tell DatePicker to do its thing then it's better to keep it undefined. Having the duplicate logic here, could cause problem in future, if one of them changes.